src/app/modules/main/chat_section/controller.nim

```diff
@@ -93,6 +93,15 @@ method init*(self: Controller) =
     var args = ContactArgs(e)
     self.delegate.onContactUnblocked(args.contactId)
 
+  self.events.on(SIGNAL_CHAT_UPDATE) do(e: Args):
+    var args = ChatUpdateArgsNew(e)
+
+    for chatDto in args.chats:
+      if not self.delegate.isChatExists(chatDto.id):
```

Comment:
I wonder if maybe we should update the chat if it already exists? In the case of a community chat update, does it happen with this event?
Otherwise, do "normal" chats have properties that could be updated?

Reply:
i think it's not update chat - its just event for chat which is not exist. Cos `invitation` like a `message`